### PR TITLE
EXPANDR-7181 Fix issues for AWS rule logic

### PR DESCRIPTION
**Pull request by capanw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-7181

## Description
SOC bug fixes

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_9.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#### Scripts
+
+##### AWSRecreateSG
+
+- Updated the script to fix an issue with security group rule case and add a change flag.
```
Comment (BigEasyJ) [inline Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_9.md:5]:
```suggestion
Updated the script to fix an issue with security group rule case and add a change flag.
```

**Comment by capanw** [comment]
@BigEasyJ Can you review this?

**Comment by capanw** [comment]
@adi88d PR is ready for review.

File: Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_9.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#### Scripts
+
+##### AWSRecreateSG
+
+Updated the script to fix an issue with security group rule case and add a change flag.
```
Comment (adi88d) [inline Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_9.md:5]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.13.82467*.
- Updated the script to fix an issue with security group rule case and add a change flag.
```

**Review by adi88d — CHANGES_REQUESTED** [review]
Great job!

**Comment by capanw** [comment]
@adi88d. Thank you for the suggestion. Updated Readme.

**Comment by ShirleyDenkberg** [comment]
@BigEasyJ @adi88d Doc review completed.

**Review by adi88d — APPROVED** [review]

**Pull request merged**